src/index.html

```diff
@@ -11,6 +11,44 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      class="container"
+      data-qa="big">
+      <label>
+        <img
+          src="./images/Search.svg"
+          alt="img-req-alt"
+          class="bigImg img"
+        >
+        <input
+          type="text"
+          name="bSearch"
+          data-qa="keypress"
+          class="form big"
```

Comment:
you did not yet learn it, so you can disregard this for now. But you want your naming to be maximally clear, so it was easy to understand what the classname is related to. It is not quite clear what is `small`
```suggestion
          class="form form__big"
```